code/modules/clothing/glasses/meson.dm

```diff
@@ -1,20 +0,0 @@
-
-//meson goggles
-
-/obj/item/clothing/glasses/meson
-	name = "optical meson scanner"
-	desc = "Used to shield the user's eyes from harmful electromagnetic emissions, also used as general safety goggles. Not adequate as welding protection."
-	icon_state = "meson"
-	item_state = "meson"
-	deactive_state = "degoggles_meson"
-	actions_types = list(/datum/action/item_action/toggle)
-	toggleable = 1
-	darkness_view = 2
-	vision_flags = SEE_TURFS
-	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
```

Comment:
why arent you just removing the lighting alpha changes so you can only see as far as the light shines